dbsync.py

```diff
@@ -240,6 +241,35 @@ def create_mergin_client():
         raise DbSyncError("Mergin Maps client error: " + str(e))
 
 
+def revert_local_changes(mc, mp, local_changes=None):
+    """Revert local changes from the existing project."""
+    if local_changes is None:
+        local_changes = mp.get_push_changes()
+    if not any(local_changes.values()):
+        return local_changes
+    print("Reverting local changes: " + str(local_changes))
+    for add_change in local_changes["added"]:
+        added_file = add_change["path"]
+        added_filepath = os.path.join(mp.dir, added_file)
+        os.remove(added_filepath)
+    for update_delete_change in chain(local_changes["updated"], local_changes["removed"]):
+        update_delete_file = update_delete_change["path"]
+        update_delete_filepath = os.path.join(mp.dir, update_delete_file)
+        delete_file = os.path.isfile(update_delete_filepath)
+        if update_delete_file.lower().endswith(".gpkg"):
+            update_delete_filepath_base = os.path.join(mp.meta_dir, update_delete_file)
+            if delete_file:
+                os.remove(update_delete_filepath)
+            shutil.copy(update_delete_filepath_base, update_delete_filepath)
+        else:
+            if delete_file:
+                os.remove(update_delete_filepath)
+            mc.download_file(mp.dir, update_delete_file, update_delete_filepath)
```

Comment:
Do we always want to download latest version of file? 

Also we should catch for errors here.

Reply:
Done.